srcpkgs/dialog/template

```diff
@@ -3,18 +3,29 @@ pkgname=dialog
 _distver=1.3
 _date=20210117
 version="${_distver}.${_date}"
-revision=1
+revision=2
 wrksrc="${pkgname}-${_distver}-${_date}"
 build_style=gnu-configure
-configure_args="--with-ncursesw --disable-nls"
-makedepends="ncurses-devel"
+configure_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool
+ --with-ncursesw --with-libtool --disable-nls"
+make_build_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
+make_install_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
```

Comment:
I'm not in love with the `LIBTOOL` part, was it fully necessary? What errors did you get?

Also, please create a `_libtool=` variable to avoid repeating the path multiple times.

Reply:
@ericonr me too, but I've tried different options without much luck.
See https://github.com/void-linux/void-packages/pull/26339#issuecomment-730720567 and other comments in this PR for more context.
I've created `_libtool` variable to avoid duplication.